code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -18,8 +18,8 @@
 /obj/structure/chair/examine(mob/user)
 	. = ..()
 	. += "<span class='notice'>It's held together by a couple of <b>bolts</b>.</span>"
-	if(!has_buckled_mobs())
-		. += "<span class='notice'>Drag your sprite to sit in it.</span>"
+	if(!has_buckled_mobs() && (can_buckle == 1))
```

Comment:
two things here;
1. you can just do (can_buckle)
2. when using bools avoid using 1 or 0, use TRUE and FALSE

Reply:
I was just following the precedent from the rest of the chair.dm where can_buckle is assigned values of 1 and 0 instead of TRUE and FALSE.

I have made the appropriate changes throughout chair.dm to bring it up to the TRUE/FALSE standard.